patterns/1-initial/secure-innersource.md

```diff
@@ -0,0 +1,144 @@
+
+As concrete example, which allows repositories to be discovered, even though there is no company-wide access, consider the following sharing levels:
+
+- **PUBLIC** - open source: findable and accessible for all SW developers in the world
+- **INTERNAL** - inner source: findable and accessible for all SW developers in the company
+- **RESTRICTED** - restricted source: findable for all SW developers in the company, low-friction process to gain full access.
+- **CLOSED** - closed source: only accessible to named individuals and not findable via the search engine for other SW developers in the company.
+
+Repositories with RESTRICTED sharing level are included in the [central catalog][innersource-portal] and/or fed into the central search mechanism, together with their `README.md` and some meta information. Their content, however, is not accessible without following a light-weight access request process. This can for example be contacting one of the code owners or some other request mechanism described in the `README.md` file.
+
+Notes:
+
+- This will likely have an impact on contributions due to the additional step to access the code.
+- It is feasible to use a tool that automatically adds the `README.md` and some other meta-data to an internal repository in GitHub, allowing the GitHub search feature to include this data in the search results.
+
+# Resulting Context
+
+*What is the situation after the problem has been solved? The original context is changed indirectly by way of the solution. Often this section can include discussion of the next possible Patterns/problems introduced. This section can be short in content - the solution may not introduce new problems or change much context.*
+
+Many times the InnerSource adoption in an organization will [start as an experiment][start-as-an-experiment], with a small number of SHARED repositories.
+
+If the task force (and with that the Security Team) collaborate in this early phase already, they can make joint experiences with how security-related issues can be identified and addressed. This also allows the Security Teams to build up more confidence on InnerSource from a security angle.
+
+With the increased confidence and lessons learned from this experiment, the task force faces less pushback when rolling out InnerSource to more repositories.
+
+Most importantly, the changes implemented through this pattern lead to more code being shared between teams, and closed source will no longer be the obvious default.
+
+- Definition of sharing levels for repositories, with accompanying guidance of which level to use for what type of repository contents.
+- Increased InnerSource awareness in Security team
+- Increased security awareness in engineering teams
+- Increased collaboration between ISPO, Engineering and Security teams
+- More code will be shared between teams, closed source is no longer the obvious default
+- Concrete implementation of security practices, potentially (partially) automated.
```

Comment:
These bullets have not been worked into the prose above yet.

When using parts of the Solution in this section, rather than just repeating it we want to make sure to describe how the Solution has been improving upon the Problem, and affecting/improving the (Initial) Context.

This can be hard to do at times but it helps to think through which specific change we want to see after the Solution has been put in place. We can think of this like Key Performance Indicators that show us whether the pattern had the expected impacted.

Reply:
First attempt to translate bullet points onto prose

Reply:
By adding the previously mentioned sentence, I think the bullets are covered.